### DISCUSSION TITLE
Skip logins step in Quickstart guide?

### DISCUSSION THREAD

**Discussion by becky-gilbert** [l3atbc-datadog/pushkin#82] [General] [answered]
Hi Pushkin team,
If you know you don't want a login system for your website, is it possible to skip over the ['setting up logins' step](https://languagelearninglab.gitbook.io/pushkin/getting-started/quickstart#setting-up-logins) in the Quickstart guide? I tried skipping it but then the local testing step didn't work. I think is because the login step involves setting up http://localhost as the URL, but just thought I'd check whether this should be possible. 
Cheers,
Becky

**Comment by jkhartshorne** [comment, marked as answer]
Yes. What you'll need to do is edit the `/pushkin/front-end/src/config.js` and set `useAuth` to false:

```
 useForum: false,
 useAuth: false,
 //Note that the forum won't work without authentication
```

**Reply by becky-gilbert** [reply]
Great, thanks!
